### DISCUSSION TITLE
No puedo ver los apuntes de strategy y temple method

### DISCUSSION THREAD

**Discussion by josebruzzoni** [dds-jv/foro#108] [Consultas generales] [answered]
Hola que tal, la clase pasada se habló de estas temáticas pero no pude leer el apunte por el siguiente error: _Repository unavailable due to DMCA takedown._ Hoy quise volver a ver si ya estaba disponible pero tampoco pude conseguirlo. 

Desde ya gracias.

**Comment by CorridoniMatias** [comment]
Hola! Estamos resolviendo eso, mientras tanto dejo los links por acá:
- [Strategy](https://sourcemaking.com/design_patterns/strategy)
- [Template Method](https://sourcemaking.com/design_patterns/template_method)

**Comment by gastonprieto** [comment, marked as answer]
@josebruzzoni y ahí actualice el link a un nuevo pdf.
